README.md

```diff
@@ -57,10 +38,134 @@ The following commands are for Ubuntu Server. However, equivalent commands shoul
+
+```shell
+modelardbd cloud path_to_local_data_folder s3://bucket-name
+```
+
+### Execute SQL
+ModelarDB includes a command-line client in the form of `modelardb`. To interactively execute SQL statements against a local instance of `modelardbd` through a REPL, simply run `modelardb`:
+
+```shell
+modelardb
+```
+
+If `modelardbd` is not running on the same host, the host `modelardb` should connect to must be specified:
+
+```shell
+modelardb 127.0.0.1
+```
+
+`modelardb` can also execute SQL statements from a file passed as a command-line argument:
+
+```shell
+modelardb 127.0.0.1 path_to_file_with_sql_statements.sql
+```
+
+`modelardbd` can also be queried programmatically [from many different programming languages](https://arrow.apache.org/docs/index.html) using Apache Arrow Flight. The following Python example shows how to execute a simple SQL query against `modelardbd` and process the resulting stream of data points using [`pyarrow`](https://pypi.org/project/pyarrow/) and [`pandas`](https://pypi.org/project/pandas/). A PEP 249 compatible connector is also available for Python in the form of [PyModelarDB](https://github.com/ModelarData/PyModelarDB).
+
+```python
+from pyarrow import flight
+
+flight_client = flight.FlightClient("grpc://127.0.0.1:9999")
+ticket = flight.Ticket("SELECT * FROM wind_turbine LIMIT 10")
+flight_stream_reader = flight_client.do_get(ticket)
+
+for flight_stream_chunk in flight_stream_reader:
+    record_batch = flight_stream_chunk.data
+    pandas_data_frame = record_batch.to_pandas()
+    print(pandas_data_frame)
+```
+
+### Ingest Data
+Before time series can be ingested into `modelardbd` a model table must be created. A model table is a specialized table for time series which must contain a single column with timestamps, one or more columns with fields (measurements as floating-point values), and zero or more columns with tags (metadata as strings). Model tables can be created using `CREATE MODEL TABLE` statements with the column types `TIMESTAMP`, `FIELD`, and `TAG`. For `FIELD` an error bound can optionally be specified in parentheses to enable lossy compression with a per value error bound, e.g., `FIELD(1)` creates a column with a one percent error bound. `FIELD` columns default to an error bound of zero when none is specified. `modelardb` also supports normal tables created through `CREATE TABLE` statements.
+
+As both `CREATE MODEL TABLE` and `CREATE TABLE` are just SQL statements, both types of tables can be created using `modelardb` or programmatically using Apache Arrow Flight. For example, a model table storing a simple multivariate time series with weather data collected at different wind turbines can be created as follows:
+
+```shell
+CREATE MODEL TABLE wind_turbine(timestamp TIMESTAMP, wind_turbine TAG, wind_direction FIELD, wind_speed FIELD(1))
+```
```

Comment:
should it be shown that model tables can be queried just like any other table, i.e., a simple SELECT ?

Reply:
Fixed by adding a line to the first paragraph in the Ingest Data section.